test/jest/mocks.ts

```diff
@@ -171,19 +176,28 @@ export const getInitialSendStateWithExistingTxState = (draftTxState) => ({
 });
 
 export function createMockInternalAccount({
```

Comment:
I had to partially cherry-pick commit in `develop` that converts `test/jest/mocks.js` into TypeScript, because the params types of `createMockInternalAccount` was being incompletely inferred, causing conflicting type errors downstream. Supplying the correct type was the only solution.